<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageProcessingDialog</class>
 <widget class="QDialog" name="ImageProcessingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>993</width>
    <height>647</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Processing</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Current Mask:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="maskComboBox"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QSplitter" name="splitter_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="opaqueResize">
         <bool>false</bool>
        </property>
        <property name="childrenCollapsible">
         <bool>false</bool>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QSplitter" name="splitter_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Processing Order:</string>
             </property>
            </widget>
            <widget class="QWidget" name="layoutWidget">
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QPushButton" name="deleteButton">
                <property name="text">
                 <string>Delete</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="deleteBottomButton">
                <property name="text">
                 <string>Delete Bottom</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="orderListWidget"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Processing Algorithm:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTabWidget" name="algorithmWidget">
            <property name="currentIndex">
             <number>1</number>
            </property>
            <widget class="QWidget" name="Sharpening">
             <attribute name="title">
              <string>Tab 1</string>
             </attribute>
             <widget class="QWidget" name="layoutWidget">
              <property name="geometry">
               <rect>
                <x>20</x>
                <y>20</y>
                <width>379</width>
                <height>54</height>
               </rect>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>Filters:</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout">
                 <item>
                  <widget class="QLabel" name="gaussFilterLabel">
                   <property name="text">
                    <string>Gauss Filter:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="gaussFilterAddButton">
                   <property name="text">
                    <string>Add</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </widget>
            <widget class="QWidget" name="Filtering">
             <attribute name="title">
              <string>Tab 2</string>
             </attribute>
             <widget class="QWidget" name="layoutWidget">
              <property name="geometry">
               <rect>
                <x>20</x>
                <y>20</y>
                <width>364</width>
                <height>54</height>
               </rect>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>Sharpeners:</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <item>
                  <widget class="QLabel" name="laplacianSharpenerLabel">
                   <property name="text">
                    <string>Laplacian Sharpener:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="laplacianSharpenerAddButton">
                   <property name="text">
                    <string>Add</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="applicationButton">
         <property name="text">
          <string>Application</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelButton">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
